src/cfg/structured.rs

```diff
@@ -91,3 +105,131 @@ impl StructuredBlock {
         }
     }
 }
+
+impl StructuredFunction {
+    pub fn to_function(&self) -> Function {
```

Comment:
same here- `UnstructuredFunction`?
Maybe a tradeoff with verbosity, but to me it feels confusing to specify structured but not unstructured (esp because the `Vec<StructuredFunctions>` field in `StructuredProgram` is named `functions`, not `structured_functions` or something)